### DISCUSSION TITLE
Is oh-my-posh works only in high end pc?

### DISCUSSION THREAD

**Discussion by victim0258** [JanDeDobbeleer/oh-my-posh#1225] [Troubleshoot] [answered]
Hello everyone,

I want to use/install oh-my-posh in the wsl2 but my PC is average. So, I would like to ask two below things before installing:-
1. Is it consumes more power/CPU usage?
2. Is it works perfectly in every PC/device?

Thank you in advance :)

**Comment by JanDeDobbeleer** [comment, marked as answer]
It works everywhere. Shouldn't be an issue. We have this running Raspberry PI's too.
